libevmcore/CommonSubexpressionEliminator.cpp

```diff
@@ -230,26 +343,40 @@ AssemblyItems CSECodeGenerator::generateCode(
 
 	// @todo: provide information about the positions of copies of class elements
 
-	// generate the dependency graph
+	// generate the dependency graph starting from final storage and memory writes and target stack contents
```

Comment:
Being extremely pedantic with the comments I would say:

// generate the dependency graph starting from final storage, memory writes and target stack contents